src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue

```diff
@@ -25,6 +25,7 @@
         <div class="text-xs font-semibold pl-6">{{$t('multisig.manageCosignatories')}}</div>
         <div class='pl-6'>
            <div class=" error error_box mb-5 whitespace-pre" v-if="err!=''">{{ err }}</div>
+           <div class="error error_box mb-5" v-if="publicKeyNotExist!=''">{{ publicKeyNotExist }}</div>
```

Comment:
not sure which public key has doesnt exist, and it's better to use yellow color to indicate warning, red is for error